### PR TITLE
feat: upgrade vuetify

### LINKED ISSUE #295: Create with Vuetify version 2.0.0
Hi all,

I've been trying to create nuxt application using Vuetify 2.0.0.

I use create-nuxt-app then try to upgrade Vuetify and modify webpack but got nowhere.
I keep getting `sass` and `node-sass` error even though i specifically added 
````
    config.module.rules.push({
        test: /\.s(c|a)ss$/,
        use: [
          'vue-style-loader',
          'css-loader',
          {
            loader: 'sass-loader',
            options: {
              implementation: require('sass'),
              fiber: require('fibers'),
              indentedSyntax: true // optional
            }
          }
        ]
      })
````

Anyone done this yet?

### PR DESCRIPTION
**Pull request by ricardogobbosouza**
Upgrade vuetify module to v1

Resolve #295 
Resolve https://github.com/nuxt-community/vuetify-module/issues/66

### CODE REVIEW

File: template/frameworks/vuetify/pages/index.vue [hunk 1]
```diff
@@ -9,7 +9,7 @@
       sm8
       md6
     >
-      <div class="text-xs-center">
+      <div class="text-center">
```
Comment (kevinmarrec) [inline template/frameworks/vuetify/pages/index.vue]: just `text-center` (without `class=`) also works :)

Comment (kevinmarrec) [reply template/frameworks/vuetify/pages/index.vue]: Nvm

**Comment by ricardogobbosouza** [comment]
@kevinmarrec i tested the generated project and it works

**Comment by Lecraminos** [comment]
When populating a vanilla project with your changes, I receive some dependency warnings:

> warning "@nuxtjs/vuetify > vuetify@2.0.0" has unmet peer dependency "vue@^2.6.4".
warning "@nuxtjs/vuetify > vuetify-loader@1.2.2" has unmet peer dependency "vue-template-compiler@^2.5.0".
warning "@nuxtjs/vuetify > vuetify-loader@1.2.2" has incorrect peer dependency "vuetify@^1.3.0".

Furthermore, it seems that e.g. the "prepend-icon" property on a v-text-field is not handled correctly, whereas the v-icon is working correctly.

**Comment by clarkdo** [comment]
@Lecraminos Looks like it's a bug from `yarn` https://github.com/yarnpkg/yarn/issues/5347, the warning is due to different version in modules:

Like nuxt -> `vue@^2.6.10` and vuetify depends `vue@^2.6.4` which are actually correct since they both support 2.6.10

**Comment by bujji1** [comment]
Looks like some issue with the template . Please see the below error with latest version of Vuetify ( 2.0.3 ) and "nuxt": "^2.8.1",

```
ERROR  Could not compile template \\node_modules\\@nuxt\\vue-app\\template\\App.js: Cannot resolve "vuetify/src/stylus/main.styl" from "\vuetify\src\stylus\main.styl"

  at Promise.all.templateFiles.map (node_modules\@nuxt\builder\dist\builder.js:5945:17)
```

**Comment by ricardogobbosouza** [comment]
@bujji1 vuetify 2 uses sass not styl

**Comment by bujji1** [comment]
thank you @ricardogobbosouza  for your response . yes , but upgrading to the latest throwing the above error .

Here are the steps to reproduce the above issue

```
1) npx create-nuxt-app sample1
2) ncu -u   ( To get the latest version of packages , You can install ncu using npm i -g npm-check-updates) 
3) npm install
4) npm run dev 
```

You will get the above issue . Thanks for checking .

**Comment by ricardogobbosouza** [comment]
@bujji1 it works, please check version of `create-nuxt-app`

**Comment by bujji1** [comment]
@ricardogobbosouza 

With `yarn create nuxt-app <my-project>`
It worked just perfect  . With yarn command it is using "@nuxtjs/vuetify": "^1.1.0",  But with npm it is not

**Comment by ricardogobbosouza** [comment]
@bujji1  what is the result?
```bash
npx create-nuxt-app --version
```

**Comment by bujji1** [comment]
Thank You @ricardogobbosouza 

`create-nuxt-app/2.9.0 win32-x64 node-v10.15.1`

Here is the package generated with npm command 
```
  "dependencies": {
    "nuxt": "^1.0.0",
    "vuetify": "^0.17.3",
    "@nuxtjs/axios": "^5.0.0"
  },
  "devDependencies": {
    "cross-env": "^5.0.1",
    "stylus": "^0.54.5",
    "stylus-loader": "^3.0.1"
  }
```

with yarn

```
   "dependencies": {
    "nuxt": "^2.8.1"
  },
  "devDependencies": {
    "@nuxtjs/vuetify": "^1.1.0",
    "nodemon": "^1.19.1"
  }
```

**Comment by bujji1** [comment]
Here is the log from npm and there are 9 vulnerabilities  . The windows command is blocking and not proceeding any further after the last statement .  After waiting for few minutes I am killing the process there and re running npm install again .


```
C:\Bujji\mygit\prod\chibu\client2>npx create-nuxt-app tk4
> Generating Nuxt.js project in C:\Bujji\mygit\prod\chibu\client2\tk4
? Project name tk4
? Project description My super-excellent Nuxt.js project
? Use a custom server framework none
? Use a custom UI framework vuetify
? Choose rendering mode Universal
? Use axios module yes
? Use eslint no
? Author name Bujji
? Choose a package manager npm
Initialized empty Git repository in C:/Bujji/mygit/prod/chibu/client2/tk4/.git/
npm WARN deprecated extract-text-webpack-plugin@3.0.2: Deprecated. Please use https://github.com/webpack-contrib/mini-css-extract-plugin
npm WARN deprecated postcss-cssnext@3.1.0: 'postcss-cssnext' has been deprecated in favor of 'postcss-preset-env'. Read more at https://moox.io/blog/deprecating-cssnext/
npm WARN deprecated browserslist@2.11.3: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
npm WARN deprecated bfj-node4@5.3.1: Switch to the `bfj` package for fixes and new features!
npm WARN deprecated flatten@1.0.2: I wrote this module a very long time ago; you should use something else.
npm WARN deprecated browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.

> core-js@2.6.9 postinstall C:\Bujji\mygit\prod\chibu\client2\tk4\node_modules\core-js
> node scripts/postinstall || echo "ignore"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon:
> https://opencollective.com/core-js
> https://www.patreon.com/zloirock

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)


> uglifyjs-webpack-plugin@0.4.6 postinstall C:\Bujji\mygit\prod\chibu\client2\tk4\node_modules\webpack\node_modules\uglifyjs-webpack-plugin
> node lib/post_install.js


> nuxt@1.4.5 postinstall C:\Bujji\mygit\prod\chibu\client2\tk4\node_modules\nuxt
> opencollective postinstall || exit 0


     *** Thank you for using nuxt! ***

Please consider donating to our open collective
     to help us maintain this package.

  https://opencollective.com/nuxtjs/donate

                    ***

npm WARN rollback Rolling back readable-stream@2.3.6 failed (this is probably harmless): EPERM: operation not permitted, scandir 'C:\Bujji\mygit\prod\chibu\client2\tk4\node_modules\fsevents\node_modules'
npm WARN rollback Rolling back node-pre-gyp@0.12.0 failed (this is probably harmless): EPERM: operation not permitted, lstat 'C:\Bujji\mygit\prod\chibu\client2\tk4\node_modules\fsevents\node_modules'
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN ajv-keywords@3.4.1 requires a peer of ajv@^6.9.1 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})


added 1113 packages from 675 contributors and audited 10495 packages in 62.531s
found 9 vulnerabilities (5 moderate, 4 high)
  run `npm audit fix` to fix them, or `npm audit` for details
```

**Comment by bujji1** [comment]
@ricardogobbosouza . It is my issue .
Some how there were multiple create-nuxt-app installed on my machine . I ran the below command and it cleaned the stuff .
`  npm uninstall -g create-nuxt-app`

Now it is working just perfect even with npm . sorry for the wrong call .

**Pull request merged**